lib/os.js

```diff
@@ -285,9 +282,6 @@ module.exports = {
   type: getOSType,
   userInfo,
   uptime: getUptime,
-
-  // Deprecated APIs
-  tmpDir: deprecate(tmpdir, tmpDirDeprecationMsg, 'DEP0022')
```

Comment:
With this line removed, the file fails linting because `deprecate` is defined but never used.